tests/functional/a/arguments_differ.py

```diff
@@ -1,5 +1,5 @@
 """Test that we are emitting arguments-differ when the arguments are different."""
-# pylint: disable=missing-docstring, too-few-public-methods, unused-argument,useless-super-delegation, useless-object-inheritance
+# pylint: disable= arguments-renamed, missing-docstring, too-few-public-methods, unused-argument,useless-super-delegation, useless-object-inheritance

```

Comment:
What is the new result of this file if we keep it the same than before? It's just to picture easily what changed in this MR :)  (thinking is hard)

Reply:
I just added that so that the test emit only the argument-differ errors. I think it is less complicated and easier to understand the test output for arguments-differ.